### PR TITLE
Update: Improve translations for importing actions with 'zh_CN'.

### PR DESCRIPTION
**Pull request by celaraze**
-  Translation text modifications.

### CODE REVIEW

File: packages/actions/resources/lang/zh_CN/import.php [hunk 1]
```diff
@@ -42,12 +42,8 @@
 
             'title' => '导入完成',
 
-            'actions' => [
-
-                'download_failed_rows_csv' => [
-                    'label' => '下载失败的行记录导入信息|下载失败的行记录导入信息',
-                ],
-
+            'download_failed_rows_csv' => [
```
Comment (zepfietje) [inline packages/actions/resources/lang/zh_CN/import.php]: Removal of the actions key doesn't seem right.

**Review by zepfietje — CHANGES_REQUESTED** [review]

File: packages/actions/resources/lang/zh_CN/import.php (hunk 1 above)
Comment (celaraze) [reply packages/actions/resources/lang/zh_CN/import.php]: I understand you. But actually, I found that the translation key of `download_failed_rows_csv` is not in the`actions`, I just adjusted it to the correct position.

Comment (zepfietje) [reply packages/actions/resources/lang/zh_CN/import.php]: Oh, thanks for noticing. Fixed in https://github.com/filamentphp/filament/commit/f9fc29920394ccd4551f7468271a44c5e1ae8366.

Could you revert that change in this PR?

Comment (celaraze) [reply packages/actions/resources/lang/zh_CN/import.php]: LOL, Okay. Sorry I haven't saw it. THX~~

Comment (celaraze) [reply packages/actions/resources/lang/zh_CN/import.php]: BTW, Do we have a code formatting tool for our project? like `./vendor/bin/pint` or `StyleCI`

Comment (zepfietje) [reply packages/actions/resources/lang/zh_CN/import.php]: Yes, Pint.

Comment (zepfietje) [reply packages/actions/resources/lang/zh_CN/import.php]: Please run it :)

Comment (celaraze) [reply packages/actions/resources/lang/zh_CN/import.php]: okay, got u. I will close the PR and make another one if it's necessary.

Comment (zepfietje) [reply packages/actions/resources/lang/zh_CN/import.php]: The other changes were just fine, right?

**Pull request closed without merging**